src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -1454,6 +1454,10 @@ private BoundExpression GetDefaultParameterValue(SyntaxNode syntax, ParameterSym
                 defaultValue = MakeConversionNode(defaultValue, parameterType, @checked: false, acceptFailingConversion: true);
             }
 
+            // CONSIDER: it feels like determining the exact value that will be emitted for a
```

Comment:
note: this would probably be a fairly difficult/substantial change and not something we would do at this phase in the release cycle.

Reply:
Agreed, we should probably open an issue to follow up.

---
In reply to: [477744390](https://github.com/dotnet/roslyn/pull/47171#discussion_r477744390) [](ancestors = 477744390)

Reply:
Opened #47352